docs/assets/sql-seed.md

```diff
@@ -0,0 +1,54 @@
+# SQL Seed Assets
+Bruin supports running SQL assets against a variety of data platforms natively. Along with SQL assets, Bruin also supports running SQL Seed assets that will populate a table with data from a csv file. 
+
+You can define SQL Seed assets in a file ending with `.yaml`:
```

Comment:
```suggestion
You can define seed assets in a file ending with `.yaml`:
```

Reply:
I think we should also add more details about the `path` parameter, since it is supposed to be optional.